docs/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table.md

```diff
@@ -0,0 +1,162 @@
+      c_currency_id VARCHAR(32)         	        NOT NULL,
+      validfrom     TIMESTAMP WITHOUT TIME ZONE    NOT NULL,
+        CONSTRAINT ht_salary_isactive_check CHECK (isactive = ANY (ARRAY['Y'::bpchar, 'N'::bpchar])),
+        CONSTRAINT ht_salary_key PRIMARY KEY (ht_salary_id),
+        CONSTRAINT ht_salary_ad_org FOREIGN KEY (AD_ORG_ID) REFERENCES AD_ORG (ad_org_id),
+        CONSTRAINT ht_salary_ad_client FOREIGN KEY (AD_CLIENT_ID) REFERENCES AD_CLIENT (ad_client_id),
+        CONSTRAINT ht_salary_c_bpartner FOREIGN KEY (C_BPARTNER_ID) REFERENCES C_BPARTNER (c_bpartner_id),
+        CONSTRAINT ht_salary_c_currency FOREIGN KEY (C_CURRENCY_ID) REFERENCES C_CURRENCY (c_currency_id)
+    );
+```
+
+### Registering the table within the Application Dictionary
+
+The following steps register the newly created table within the Etendo Classic Application Dictionary.
+
+For this purpose, first log into Etendo Classic using a username with access to `System Administrator` > `role`. Navigate to `Application Dictionary`> `Tables and Columns` and create a new record as shown in the screenshot below:
+
+![](/assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table-1.png)
+
+Main fields of this window are (for more information, see the AD_Table table description):
+
+- _Data Package_ specifies to which java data package within the module the table will belong when used within [DAL](../../../developer-guide/etendo-classic/concepts/Data_Access_Layer.md) (Data Access Layer).
+- _Name_ Defines the name that Etendo Classic uses to recognize the defined database table. This name is used in [REST webservices](../../../developer-guide/etendo-classic/concepts/XML_REST_Web_Services.md) and in the [Data Access Layer](../../../developer-guide/etendo-classic/concepts/Data_Access_Layer.md#entity-naming).
+- _Description_ Gives a small description of the table.
+- _Help/Comments_ Defines the text that is displayed in Help window.
+- _DB Table name_ Defines database table name as it was defined by the CREATE TABLE during its creation.
+- _Java Class Name_ This will be the actual Java class within the Data Package of the module through which you will be able to access this table when using DAL.
+- _Data Access Level_ determines what kind of data will the table contain due to the multitenancy functionality
+  - _System only_: Only system records can be inserted into this table (AD_CLIENT_ID=0, AD_ORG_ID=0), for example AD_TABLE.
+  - _System/Client_: System or client specific records can be inserted here (AD_CLIENT_ID=anything, AD_ORG_ID=0), for example AD_ROLE
+  - _Organization_: Only client and organization specific data can be inserted into this table (AD_CLIENT_ID<>0, AD_ORG_ID<>0), for example C_INVOICE
+  - _Client/Organization_: Only client specific data can be inserted here, however, it can belong to a specific organizations within that client or be shared among all (AD_CLIENT_ID<>0,AD_ORG_ID=anything), for example C_BPARTNER
```

Comment:
```suggestion
  - _Client/Organization_: Only client specific data can be inserted here, however, it can belong to a specific organizations within that client or be shared among all (AD_CLIENT_ID<>0,AD_ORG_ID=anything), for example C_BPARTNER
```
```suggestion
  - _Client/Organization_: Only client specific data can be inserted here, however, it can belong to a specific organizations within that client or be shared among all (AD_CLIENT_ID<>0,AD_ORG_ID=anything), for example C_BPARTNER
```